lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -280,15 +280,12 @@ private function reverseEngineerMappingFromDatabase(): void
             return;
         }
 
-        $tables = [];
-
-        foreach ($this->_sm->listTableNames() as $tableName) {
-            $tables[$tableName] = $this->_sm->listTableDetails($tableName);
-        }
+        $tables = $this->_sm->listTables();
 
         $this->tables = $this->manyToManyTables = $this->classToTableNames = [];
 
-        foreach ($tables as $tableName => $table) {
+        foreach ($tables as $table) {
```

Comment:
Let's get rid of it and inline it here if that's indeed the case?

```suggestion
        foreach ($this->_sm->listTables() as $table) {
```